lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveRecord::Acts::List::SequentialUpdatesDefiner #:nodoc:
+  def self.call(caller_class, column, sequential_updates_option)
+    caller_class.class_eval do
+      @sequential_updates = nil
+
+      define_method :sequential_updates? do
```

Comment:
Ah I see :) Sorry about that :) I was focused on the instance variable and didn't realise you were talking about the incoming config :) I agree, no class variables is definitely preferred.